package.json

```diff
@@ -11,6 +11,8 @@
     "@oclif/core": "^1.2.0",
     "@asyncapi/diff": "^0.3.0",
     "@asyncapi/parser": "^1.14.0",
+    "@oclif/plugin-help": "^5.1.11",
```

Comment:
```suggestion

```

We don't need `@oclif/plugin-help` because it is now supported out of the box.  https://oclif.io/docs/help_classes

Reply:
Right !